chain/beacon/sync_manager.go

```diff
@@ -219,22 +259,41 @@ func (s *SyncManager) tryNode(global context.Context, upTo uint64, peer net.Peer
 				return false
 			}
 
-			logger.Debugw("new_beacon_fetched",
-				"with_peer", peer.Address(),
-				"from_round", last.Round+1,
-				"got_round", beaconPacket.GetRound())
+			// We rate limit our logging, but when we are "close enough", we display all logs in case we want to follow
+			// for a long time.
+			if idx := beaconPacket.GetRound(); target < idx || target-idx < commonutils.RateLimit || idx%commonutils.RateLimit == 0 {
+				logger.Debugw("new_beacon_fetched",
+					"with_peer", peer.Address(),
+					"from_round", from,
+					"got_round", idx)
+			}
+
 			beacon := protoToBeacon(beaconPacket)
 
 			// verify the signature validity
 			if err := s.verifier.VerifyBeacon(*beacon, s.info.PublicKey); err != nil {
-				logger.Debugw("invalid_beacon", "with_peer", peer.Address(), "round", beacon.Round, "err", err, "beacon", fmt.Sprintf("%+v", beacon))
+				logger.Debugw("Invalid_beacon", "from_peer", peer.Address(), "round", beacon.Round, "err", err, "beacon", fmt.Sprintf("%+v", beacon))
 				return false
 			}
 
-			if err := s.store.Put(beacon); err != nil {
-				logger.Debugw("unable to save", "with_peer", peer.Address(), "err", err)
-				return false
+			if force {
+				logger.Debugw("Using ForcePut to save beacon", "beacon", beacon.Round)
+				if err := s.store.ForcePut(beacon); err != nil {
```

Comment:
Maybe we could solve my dilemna with a vocabulary change: what about `Replace` ?  Force seems to imply we are skipping all checks etc while we shouldn't.

Reply:
We kinda are, we are skipping checks from some of the other Stores, such as the AppendStore. 
But I'm fine with Replace as well.